tests/resources/noxfile_nested.py

```diff
@@ -12,8 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import nox
 
-class Namespace:
-    """A class that makes the provided kwargs available as attributes."""
-    def __init__(self, **kwargs):
-        self.__dict__.update(kwargs)
+
+def eat(cheese):
+    print("Noms, {} so good!".format(cheese))
```

Comment:
if you're just printing, you don't need this indirection.

Reply:
Yeah I saw that such calls were considered [legacy][1] now. I was just looking for an excuse to use the `session` object. Any better ideas?

[1]: https://github.com/theacodes/nox/blob/ad99797b630ca7abeb6bdf115ad0b4d788c30d26/nox/sessions.py#L158-L160

Reply:
Eh, just leave it unused?